cirq-core/cirq/ops/common_channels_test.py

```diff
@@ -498,7 +498,7 @@ def test_reset_act_on():
         target_tensor=cirq.one_hot(
             index=(1, 1, 1, 1, 1), shape=(2, 2, 2, 2, 2), dtype=np.complex64
         ),
-        available_buffer=np.empty(shape=(2, 2, 2, 2, 2)),
+        available_buffer=np.empty(shape=(2, 2, 2, 2, 2), dtype=np.complex64),
```

Comment:
If this PR adds qudit support, it should include qudit tests.

Reply:
They already exist: https://github.com/quantumlib/Cirq/blob/58f6a6f4078943193d080e20727840614fa29cc3/cirq-core/cirq/sim/sparse_simulator_test.py#L62.